README.adoc

```diff
@@ -287,12 +287,12 @@ another `@Traced` annotation.
 `inventory/src/main/java/io/openliberty/guides/inventory/InventoryManager.java`
 ----
 [role="edit_command_text"]
-Enable tracing of the `list()` non-JAX-RS method by updating [hotspot=42 file=0]`@Traced` as shown.
+Enable tracing of the `list()` non-JAX-RS method by updating [hotspot=Traced file=0]`@Traced` as shown.
 
 InventoryManager.java
-[source, Java, linenums, role="code_column"]
+[source, Java, linenums, role='code_column hide_tags=copyright']
```

Comment:
- `hide_tags` should also be hiding `customTracer`, but keep `addToInvList` line

Reply:
Addressed.

Reply:
This `custom-tracer` is still not being hidden, but keep `addToInvList` tag:
![image](https://user-images.githubusercontent.com/46586499/69678098-8a403000-1072-11ea-8248-160b2e349ed7.png).

The other thing you called `customTracer` is hidden now, which is good.

What it should look like:
![image](https://user-images.githubusercontent.com/46586499/69678254-12263a00-1073-11ea-9400-a0b8817b4851.png)
What it currently looks like:
![image](https://user-images.githubusercontent.com/46586499/69678263-194d4800-1073-11ea-8f04-ffda60c1c995.png)

Reply:
fixed.